homeassistant/config.py

```diff
@@ -559,6 +559,9 @@ def _log_pkg_error(package: str, component: str, config: Dict, message: str) ->
 
 def _identify_config_schema(module: ModuleType) -> Optional[str]:
     """Extract the schema and identify list or dict based."""
+    if not hasattr(module.CONFIG_SCHEMA, "schema"):  # type: ignore
```

Comment:
Would it be a better check if we do
```suggestion
    if not isinstance(module.CONFIG_SCHEMA, vol.Schema):
```